src/components/transport_manager/test/transport_manager_impl_test.cc

```diff
@@ -637,22 +638,39 @@ TEST_F(TransportManagerImplTest, UpdateDeviceList_AddNewDevice) {
   device_list_.pop_back();
 }
 
+struct DeviceInfoComparator {
+    const transport_manager::DeviceInfo& val_;
```

Comment:
@LuxoftAKutsan @okozlovlux Data members of structs, both static and non-static, are named like ordinary nonmember variables. They do not have the trailing underscores that data members in classes have.